### PR TITLE
Add error messages for missing fields and resolve NumberFormatException

### LINKED ISSUE #214: Fix NumberFormatException from Rating class
For commands `RAddCommand` and `REditCommand` , if the `Rating ` CLI syntax is given but no value is provided eg. `r/ `, a `NumberFormatException `is thrown and not handled properly.

### LINKED ISSUE #215: Add error messages for missing fields in RAddCommand and SAddCommand

### PR DESCRIPTION
**Pull request by Hikoya**
Resolves #214 
Resolves #215

### CODE REVIEW

File: src/test/java/foodwhere/model/review/RatingTest.java [hunk 1]
```diff
@@ -25,11 +25,15 @@ public void isValidRating() {
...
 
         // invalid contents
-        assertFalse(Rating.isValidRating(6)); // invalid number
-        assertFalse(Rating.isValidRating(7)); // invalid number
+        assertFalse(Rating.isValidRating("asdf")); // invalid number
+        assertFalse(Rating.isValidRating("7")); // invalid number
+        assertFalse(Rating.isValidRating("8")); // invalid number
+        assertFalse(Rating.isValidRating("")); // invalid number
+        assertFalse(Rating.isValidRating("abcd")); // invalid number
+        assertFalse(Rating.isValidRating("   ")); // invalid number
```
Comment (clarence-chew) [inline src/test/java/foodwhere/model/review/RatingTest.java:33]: I think I would check "03" (I would think valid?), "3." (probably invalid?), "3.0" (probably invalid?), "3 ", "3a", " 3", "a3", "-3", "-10000000000000000000", "10000000000000000000", "1180591620717411303427" (overflow cases and 2^70+3) too.

File: src/main/java/foodwhere/model/review/ReviewBuilder.java [hunk 2]
```diff
@@ -30,7 +30,7 @@ public ReviewBuilder() {
         name = new Name(DEFAULT_NAME);
         date = new Date(DEFAULT_DATE);
         content = new Content(DEFAULT_CONTENT);
-        rating = new Rating(DEFAULT_RATING);
+        rating = new Rating(String.valueOf(DEFAULT_RATING));
```
Comment (clarence-chew) [inline src/main/java/foodwhere/model/review/ReviewBuilder.java:33]: Perhaps we can store DEFAULT_RATING as a String?

**Review by clarence-chew — APPROVED** [review]
Approved with concerns.

File: src/test/java/foodwhere/model/review/RatingTest.java (hunk 1 above)
Comment (Hikoya) [reply src/test/java/foodwhere/model/review/RatingTest.java:33]: Resolved in latest commit

File: src/main/java/foodwhere/model/review/ReviewBuilder.java (hunk 2 above)
Comment (Hikoya) [reply src/main/java/foodwhere/model/review/ReviewBuilder.java]: Resolved in latest commit

**Pull request merged**